tests/main/views/test_buyers.py

```diff
@@ -3598,7 +3598,7 @@ def teardown_method(self, method):
         self.data_api_client_patch.stop()
         super(TestCancelBrief, self).teardown_method(method)
 
-    def test_cancel_brief_200s_with_correct_default_content(self):
+    def test_cancel_brief_form_displays_correctly(self):
```

Comment:
Can we adjust this test name to something like `test_cancel_brief_form_displays_correctly_accessed_from_brief_overview_link` to be consistent with the one below?

Reply:
Done :+1: